packages/wrangler/src/__tests__/r2.test.ts

```diff
@@ -106,7 +106,10 @@ describe("r2", () => {
 			  -c, --config   Path to .toml configuration file  [string]
 			  -e, --env      Environment to use for operations and .env files  [string]
 			  -h, --help     Show help  [boolean]
-			  -v, --version  Show version number  [boolean]"
+			  -v, --version  Show version number  [boolean]
+
+			Options:
+			  -r, --region  A hint to control where the bucket will be created  [string] [choices: \\"auto\\", \\"WNAM\\", \\"ENAM\\", \\"WEUR\\", \\"APAC\\", \\"OC\\", \\"EEUR\\", \\"SAM\\", \\"AFR\\", \\"ME\\", \\"CHINA\\"]"
```

Comment:
Who would I get to review this copy? A technical writer, or are y'all responsible?

Reply:
Normally the product's PM chimes in with their thoughts